package.json

```diff
@@ -36,6 +34,9 @@
     "should": "^11.1.2",
     "standard-version": "^4.0.0"
   },
+  "peerDependencies": {
+    "webpack": "^3.0.0 || ^4.0.0"
```

Comment:
new major versions of webpack-contrib and webpack org modules are only supporting webpack@4 moving forward. since this is a new major version, we should drop webpack@3 support.

Reply:
Agree with Andrew, it's harder to keep compat when things move forward too

Reply:
@shellscape :+1: